rfc-097-verify-specific-start-pages.md

```diff
@@ -0,0 +1,51 @@
+# Providing alternative GOV.UK service start pages for the GOV.UK Verify single IDP journey
+
+## Summary
+
+This document describes an approach for providing alternative service start pages 
+that will be displayed to users on a GOV.UK Verify single IDP journey.
+
+## Terms
+
+* IDP - Identity Provider (also known as a Certified Company)
+* RP - Relying Party (also known as a transaction or service, e.g. view your driving license or check your state pension)
+
+## Problem
+
+We have recently introduced a new type of journey to Verify. This journey type allows Identity Providers (IDPs) to target their existing customers in order to drive customers into using their identity service with GOV.UK services. Unlike traditional Verify journeys, the user will actually start on a page at, or an e-mail from, a specific IDP where they will choose which service to use. During this journey Verify restricts their ability to choose an alternative IDP. This is know as the Single IDP journey and is described in Verify [RFC-041](https://github.com/alphagov/verify-architecture/blob/master/rfcs/rfc-041-single-idp-journey.md).
+
+When following a single IDP journey, the user needs to be redirected via the RP (service) in order to pick up a valid SAML request. Initial thoughts were to direct the RP headless start page, which generates the SAML request and results in an immediate redirect back to the Verify hub.
+This approach, however, is not desirable as there is often important information given to the
+user on the service's normal start page. Similarly, showing the normal start page is not
+desirable as it may give service sign-in options that would direct the user away from Verify and the
+user's chosen IDP (for example, if the user chooses to login with Government Gateway).
+
+The approach described in this document is to implement alternate service start pages, that
+will display all important information to the user but only give them the option of using
+Verify and, hence, their chosen IDP. 
+
+### Considerations
+
+#### Discoverability
+
+Any customised content is designed to be reachable solely via hyperlinks from IDPs, which will redirect via the Verify hub.
```

Comment:
is this accurate? the customised content will be only reached via redirects from hub. Only the hub knows the URLs.

Reply:
We can handle this in the frontend by serving robots tags for search engines. For internal search, this is only one content item so only the "parent" start page would be indexed by default.